lib/DDGC/DB/ResultSet/ReleaseVersion.pm

```diff
@@ -0,0 +1,20 @@
+package DDGC::DB::ResultSet::ReleaseVersion;
+# ABSTRACT: Resultset class for tracking release versions 
+
+use Moose;
+extends 'DDGC::DB::Base::ResultSet';
+use namespace::autoclean;
+use DateTime;
+use DateTime::Format::Pg;
+
+sub last_modified {
```

Comment:
This is in the base ResultSet class so shouldn't need to be defined here (that change just went in on Friday)